tests/MudBlazor.Markdown.Tests/MudMarkdownTestsBase.cs

```diff
@@ -8,11 +8,19 @@ public abstract class MudMarkdownTestsBase : IDisposable
 	{
 		private readonly TestContext _ctx = new();
 
-		protected IRenderedComponent<MudMarkdown> CreateFixture(string value, ICommand command = null, int? tableCellMinWidth = null) =>
+		protected IRenderedComponent<MudMarkdown> CreateFixture(string value, ICommand command = null, int? tableCellMinWidth = null,
```

Comment:
non nullable default values do not verify default values of the actual component
1) create an extension method which does not set null values (file ComponentParameterCollectionBuilderEx)
```cs
public static ComponentParameterCollectionBuilder<TComponent> AddIfNotNull<TComponent, TValue>(
	this ComponentParameterCollectionBuilder<TComponent> @this,
	Expression<Func<TComponent, TValue>> parameterSelector,
	TValue value)
	where TComponent : IComponent
{
	return value != null
		? @this.Add(parameterSelector, value)
		: @this;
}
```
2) use the extension method for `Typo`
```cs
protected IRenderedComponent<MudMarkdown> CreateFixture(Typo? h1Type = null, Typo? h2Type = null, etc.)
```
```cs
.AddIfNotNull(static x => x.H1Typo, h1Typo) etc.
```
3) keep `Typo h6Typo = Typo.h6` in the line 12?

Reply:
`null` is not the parameter, but ok I will fix it in the next commit